src/components/Book.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Addbook from './Addbook';
+
+export default class Book extends Component {
+  render() {
+    let Book = this.props.books
+    console.log(Book)
```

Comment:
[OPTIONAL]
- [ ] Kindly remember to remove console logs from your codebase. In bigger applications, can cause unexpected issues like unintended leakage of confidential data.